prototype_source/gpu_direct_storage.py

```diff
@@ -0,0 +1,126 @@
+"""
+(prototype) Using GPUDirect Storage
```

Comment:
I know next to nothing about SEO, but it feels like one can discover this tutorial only if user knows a very specific technical term. Shouldn't it be something like "Accelerating data loading with GPU Direct Storage"?